src/services/auth/index.tsx

```diff
@@ -11,15 +10,22 @@ export class LoginError extends Error {
   }
 }
 
+export class LoginLockedOutError extends LoginError {
```

Comment:
Small nit, this is a bit of a mouthful. What if we reduce it to `LockedOutError` ah? Any issues? 😆

Reply:
Sure am good with this

Reply:
it's less descriptive i guess, like i might not know right away that it is related to login. what about `LoginLockedError`?😅

Reply:
Hmm, that's true too. I'm good with `LoginLockedError` haha. Seb? ^^

Reply:
sure